### ISSUE TITLE
The `indentation` rule never measures the line a terminating semicolon stands on alone

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#569]
### What minimal example or steps are needed to reproduce the bug?

Where the semicolon closing the last statement of a block stands on a line of its own, that line is measured by nobody — the same shape as [#509](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/509), one raw over.

```css
a {
	@extend .b
			;
}
```

The parser files the whitespace in front of the semicolon into the at-rule's `raws.between`, which here is `⏎\t\t\t`, and leaves the block's `raws.after` at `⏎` with `raws.semicolon` set. The block's own reading of `raws.after` therefore measures line 4, the brace's line, and is right to; and the text `checkAtRuleParams` measures the params in runs through `raws.between` but is trimmed before the search, so the run holding line 3 is cut off its end.

The declaration spelling is silent by a different route — there the run stands in `decl.raws.value.raw`, which is `pink⏎\t\t\t`:

```css
a {
	color: pink
			;
}
```

| input | warnings |
| --- | --- |
| `a {⏎\t@extend .b⏎\t\t\t;⏎}` | none |
| `a {⏎\tcolor: pink⏎\t\t\t;⏎}` | none |
| `a {⏎\t@extend .b;⏎\t\t\t}` | `3:4 Expected indentation of 0 tabs` |

Measured on `2b04870` under `postcss`, `postcss-scss` and `postcss-less` alike, and on `cdf1784` — the base of #509's branch — with the same answers, so this is older than that fix and untouched by it.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": ["@stylistic/stylelint-plugin"],
	"rules": {
		"@stylistic/indentation": "tab"
	}
}
```

### What did you expect to happen?

The line a terminating semicolon stands on is measured like any other line of the block, and brought to its level by `--fix`.

### What actually happened?

It is measured by neither reader: the block's `raws.after` starts behind the semicolon, and the trim in front of the params search cuts the run that holds it.

### Notes

Found while closing #509. Whose line it is, is the question to settle first — the statement's, since the semicolon closes it, or the block's. Only after that does it follow which raw the fix writes to.
